### PR TITLE
Fix mockgen issue with trailing dot

### PR DESCRIPTION
**Pull request by alxsap**
The latest master version of mockgen generates the trailing periods. I have version v1.4.3 which does not generate them.

Without `GO111MODULE=on` `go get` will install the latest version from the master branch of a module. With, it is possible to specify the module release tag. This caused the issue of unexpected, different mock generations.

### CODE REVIEW

File: hack/update-codegen.sh [hunk 1]
```diff
@@ -119,6 +114,9 @@ if [[ -z $GOPATH ]]; then
 fi
 GOPATH_1=${GOPATH%%:*}  # the first entry of the GOPATH
 
+# Enable module mode to allow setting a version in 'go get'
+export GO111MODULE=on
```
Comment (anbrsap) [inline hack/update-codegen.sh:118]: You may also prefix the `go get` command directly, to make it obvious that this is the one being affected:

```go
( cd "$GOPATH_1" && GO111MODULE=on go get github.com/golang/mock/mockgen@v1.4.3 ) || die "Installation of mockgen failed"
```

File: hack/update-codegen.sh [hunk 2]
```diff
@@ -155,7 +153,8 @@ echo "CODEGEN_PKG:  $CODEGEN_PKG"
 echo "GOPATH:       $GOPATH_1"
 echo "VERIFY:       $(if is_verify_mode; then echo "true"; else echo "false"; fi)"
 echo "GO version:   $(go version)"
-
+echo "--- Environment with 'go': ---"
```
Comment (rinckm) [inline hack/update-codegen.sh:156]: Was this introduced only for debugging or should this stay?

Comment (alxsap) [reply hack/update-codegen.sh:156]: Might be helpful to know in future. But I can remove it.

Comment (rinckm) [reply hack/update-codegen.sh:156]: If you remove it you can also do the change proposed by @anbrsap, otherwise it is misleading.

Comment (alxsap) [reply hack/update-codegen.sh]: https://github.com/SAP/stewardci-core/pull/101/commits/7368698d9465ef9da71fd320219626f59490dae3

File: hack/update-codegen.sh (hunk 1 above)
Comment (alxsap) [reply hack/update-codegen.sh]: https://github.com/SAP/stewardci-core/pull/101/commits/7368698d9465ef9da71fd320219626f59490dae3

**Pull request merged**